src/Compilers/Core/Portable/Symbols/Attributes/CommonTypeWellKnownAttributeData.cs

```diff
@@ -246,5 +246,23 @@ public bool HasExcludeFromCodeCoverageAttribute
         }
 
         #endregion
+
+        #region SkipLocalsInitAttribute
+        private bool _hasSkipLocalsInitAttribute;
```

Comment:
>_hasSkipLocalsInitAttribute [](start = 21, length = 27)

Please move this to C# specific structure and do the same for other symbols

Reply:
Done.